integration-cli/docker_cli_network_unix_test.go

```diff
@@ -1128,10 +1128,10 @@ func (s *DockerNetworkSuite) TestDockerNetworkConnectDisconnectToStoppedContaine
 
 func (s *DockerNetworkSuite) TestDockerNetworkConnectPreferredIP(c *check.C) {
 	// create two networks
-	dockerCmd(c, "network", "create", "--subnet=172.28.0.0/16", "--subnet=2001:db8:1234::/64", "n0")
+	dockerCmd(c, "network", "create", "--ipv6", "--subnet=172.28.0.0/16", "--subnet=2001:db8:1234::/64", "n0")
```

Comment:
This was the docker PR: https://github.com/docker/docker/pull/17513
and this was the libnetwork part: https://github.com/docker/libnetwork/pull/912